### ISSUE TITLE
cannot build MathOptInterfaceGLPK

### ISSUE THREAD

**Issue by jd-lara** [joaquimg/MathOptInterfaceGLPK.jl#1]
I have been trying to run the example of MOI for the knapsack problem 

```julia
using MathOptInterface
const MOI = MathOptInterface
using MathOptInterfaceGLPK
```

But at the build stage I get this error in the GLPK interface 

```
using MathOptInterfaceGLPK
ERROR: LoadError: LoadError: UndefVarError: AbstractSolverInstance not defined
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:576
 [2] eval(::Module, ::Any) at ./boot.jl:235
 [3] _require(::Symbol) at ./loading.jl:490
 [4] require(::Symbol) at ./loading.jl:405
 [5] include_from_node1(::String) at ./loading.jl:576
 [6] eval(::Module, ::Any) at ./boot.jl:235
 [7] _require(::Symbol) at ./loading.jl:490
 [8] require(::Symbol) at ./loading.jl:405
while loading /Users/jdlara/.julia/v0.6/LinQuadOptInterface/src/LinQuadOptInterface.jl, in expression starting on line 98
while loading /Users/jdlara/.julia/v0.6/MathOptInterfaceGLPK/src/MathOptInterfaceGLPK.jl, in expression starting on line 13
```

**Comment by blegat** [comment]
We very recently renamed it to `AbstractOptimizer` in https://github.com/JuliaOpt/MathOptInterface.jl/pull/217

**Comment by jd-lara** [comment]
@blegat I recently made a pull request to fix it 

https://github.com/JuliaOpt/LinQuadOptInterface.jl/pull/3

**Comment by joaquimg** [comment]
I am fixing LQOI hold a little more

**Comment by joaquimg** [comment]
closed by 848fdd7b4dc45a83bd08002e6924b0c9ce394476
